.readthedocs.yml

```diff
@@ -22,10 +26,10 @@ formats:
 # Optional but recommended, declare the Python requirements required
 # to build your documentation
 # See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
-python:
-  install:
-    - method: pip
-      path: .
-      extra_requirements:
-        - docs
-        - all
\ No newline at end of file
+# python:
+#   install:
+#     - method: pip
+#       path: .
+#       extra_requirements:
+#         - docs
+#         - all
```

Comment:
Everything looks good, although small nit do we need this commented-out code if we will be handling it in the code block above?

Reply:
I thought about leaving it as a reminder to re-enable this approach once we have a more reliable way to install Spacepy. I should probably add this note as a comment in that area though so it doesn't just look like commented out code. 

We'll see what Steven thinks.

Reply:
Fine with me.